virttest/qemu_devices/qcontainer.py

```diff
@@ -1288,8 +1288,9 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus,
                 bus += ".%d" % unit
             # If bus was not set, don't set it, unless the device is
             # a spapr-vscsi device.
-            elif _bus is None and 'spapr_vscsi' not in _hba:
-                bus = None
+            elif _bus is None:
+                if 'virtio-scsi-pci' and 'spapr_vscsi' not in _hba:
```

Comment:
` if 'virtio-scsi-pci' and 'spapr_vscsi' not in _hba:` is equivalent to  ` if True and 'spapr_vscsi' not in _hba:`